### DISCUSSION TITLE
[feature request] Infill extrusion multiplier (flow)

### DISCUSSION THREAD

**Discussion by fmonaca** [OrcaSlicer/OrcaSlicer#586] [Ideas]
Hi, would it be possible to also set the infill flow like in Cura? Cheers!

**Comment by anmarcu** [comment]
do you mean to have 97% instead 0.97? 
This is inherited from the original fork and idk if it really makes sense to express it as a percentage since it's used to adjust the other flows.

**Comment by fmonaca** [comment]
I'm talking about the infill flow, I'm not sure I understand your question,.. I could not find an option to specify the flow for the infill. You can set a different flow for bridges, top layer and bottom layer, for example.

**Reply by anmarcu** [reply]
I get your point, this is similar with what SS can offer.
That would be nice to have, as Ellis explained in his profile, with higher fow for infill, even at 40% will be less lines in grid and a bit more robust.

I thought that you want to modify the input value from a number into a percentage as Cura is using.

Cheers

**Comment by knw257** [comment]
I would also like to see this, specifically because when using infill patterns that crosses over itself on the same layer (e.g. triange, tri-hex, rectalinear, etc), the nozzle tends to scrape and make a grinding noise. Research I've done suggests that lowering the flow rate of the infill can help reduce or eliminate this, so it would be really useful to reduce wear on consumables like nozzles and overall vibration.

**Comment by jstill1981** [comment]
Same here, this option would be extremely useful!
